neo4j/connection.py

```diff
@@ -70,7 +70,8 @@
 log_error = log.error
 
 
-def hex2(x):
+def hex2(str):
+    x = ord(str)
```

Comment:
Couple of things here: `str` is a reserved word in Python and this functionality varies between Python 2 and 3. Can pair on this with you :-)